data/presets/baking_oven/bakehouse.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "Bakehouse",
```

Comment:
Hi. We need a valid icon for the preset. See https://github.com/ideditor/schema-builder/blob/main/ICONS.md for more information about which icon sets are available.

As a last resort, we could use `fas-bread` (https://fontawesome.com/v6/icons/bread-slice?f=classic&s=solid) maybe?

Reply:
I have used now Temaki-bread for the icon